Library/Homebrew/dev-cmd/audit.rb

```diff
@@ -532,8 +532,10 @@ def audit_conflicts
       end
     end
 
+    versioned_conflicts_whitelist = %w[node@ bash-completion@].freeze
```

Comment:
Is there any compelling reason to move this out of the `audit_conflicts` method and turn it into a true constant?